pkg/vgmanager/vgmanager_controller.go

```diff
@@ -90,12 +91,11 @@ func (r *VGReconciler) reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		return ctrl.Result{}, err
 	}
 
-	//TODO: actually check the node against the nodeSelector.
-	node := &corev1.Node{}
-	nodeMatches, err := NodeSelectorMatchesNodeLabels(node, volumeGroup.Spec.NodeSelector)
+	//Does this node need to process it?
```

Comment:
can you elaborate a bit more?

Reply:
If the nodeSelector for the VG does not match the node the vgmanager is running on, it does not need to do anything.

Reply:
ok.  The comment is framed as a question.  Should it rather be a statement.